mathics/builtin/numbers/constants.py

```diff
@@ -239,7 +239,7 @@ class ComplexInfinity(_SympyConstant):
      = Indeterminate
     """
 
-    summary_text = "infinite complex quantity of undertermined direction"
+    summary_text = "represents an infinite complex quantity of undetermined direction"
```

Comment:
"represents an" doesn't do anything here. I mentioned this before. (The spelling correction is good though.)